myRental/modules/Log.js

```diff
@@ -0,0 +1,17 @@
+let method = logModule.prototype;
+
+let Clrlog = require('clrlog');
+
+let config = require('../config.json');
+
+let myClrlog = null;
+
+function logModule() {
```

Comment:
Since you are already using anonymous functions, this should be able to be rewritten as 
```
logModule = () => {
    myClrlog = new Clrlog("LOG", 'success', `${__dirname}/../${config.log_address}`, 'LOG:');
};
```